code/plot_amdahl.py

```diff
@@ -0,0 +1,96 @@
+"""
+Plotting script for HPC Workflows scaling study
+"""
+
+import json
+import matplotlib.pyplot as plt
+import pandas as pd
+import sys
+
+from math import ceil
+from matplotlib.lines import Line2D
+
+
+def actual_speedup(parallel_wall_time, serial_wall_time):
+    return serial_wall_time / parallel_wall_time
+
+
+def amdahl_speedup(serial_proportion, parallel_proportion, num_cores):
+    normalized_run_time = serial_proportion + (parallel_proportion / num_cores)
+    return 1.0 / normalized_run_time
+
```

Comment:
Add extra line break here for consistency (and possibly PEP 8?).

Reply:
Good catch! Fixed by bb8b626